sdk/search/azure-search-documents/tests/test_search_client_search_live.py

```diff
@@ -79,7 +81,9 @@ def _test_get_search_filter_array(self, client):
             "hotelName",
             "description",
             "@search.score",
+            "@search.reranker_score",
```

Comment:
There were no new recordings needed for this right ? The response had them already, we were just not exposing them

Reply:
Yes. You are right. These properties are already in the response. Just we did not expose them.